### ISSUE TITLE
Cannot assign to read only property 'nodeName' of object '#<HTMLHeadingElement>'

### ISSUE THREAD

**Issue by ccqgithub** [GetmeUK/ContentTools#501]
```
Cannot assign to read only property 'nodeName' of object '#<HTMLHeadingElement>'
```

Thorw that error when paste.

```
if (node.nodeName = '#text' && node.textContent.trim() === '') {
    continue;
}
```
maybe that will be `node.nodeName == '#text'` ?

**Comment by anthonyjb** [comment]
Resolved in 1.6.5

**Comment by ccqgithub** [comment]
thanks
